packages/postcss-merge-rules/src/__tests__/index.js

```diff
@@ -786,3 +786,11 @@ test(
     '.a{color:red;display:flex;font-size:10px;}.a,.c{unicode-bidi:normal;}.c{all:unset;color:red;display:flex;font-size:10px;}'
   )
 );
+
+test(
+  'should not merge :focus-visible',
+  processCSS(
+    'a{color : green;} a:focus-visible{ color : green;} a:focus-visible{ background : red}',
+    'a{color : green;} a:focus-visible{ color : green;} a:focus-visible{ background : red}'
+  )
+);
```

Comment:
Can we add test where we merge them? Here example of the test https://github.com/cssnano/cssnano/blob/5db8a8e5aa9a6a744b3f6c34dc86d992d73bf1c7/packages/postcss-merge-rules/src/__tests__/index.js#L517

Reply:
sure , 
I will add them .....

Reply:
this is not  supported fully by any browser as of now

![image](https://user-images.githubusercontent.com/26347874/75677247-fef9ba80-5cb0-11ea-9650-d1d13640d0ed.png)

Reply:
Yes, just test

Reply:
which browser version to use in order to merge it ? none of them are supporting it. So it can be merged

Reply:
Oh, yes, I see